sklearn/ensemble/tests/test_voting.py

```diff
@@ -124,6 +163,15 @@ def test_weights_iris():
     scores = cross_val_score(eclf, X, y, cv=5, scoring='accuracy')
     assert_almost_equal(scores.mean(), 0.93, decimal=2)
```

Comment:
I can see that there are similar poorly designed tests for VotingClassifier. I wish they were better.

Reply:
I think VotingClassifier tests should be fixed in separate PR.